packages/medusa/src/types/store.ts

```diff
@@ -8,3 +12,31 @@ export type UpdateStoreInput = {
   metadata?: Record<string, unknown>
   default_sales_channel_id?: string
 }
+
+/**
+ * @schema ExtendedStoreDTO
+ * allOf:
+ *   - $ref: "#/components/schemas/Store"
+ *   - type: object
+ *     required:
+ *       - payment_providers
+ *       - fulfillment_providers
+ *       - feature_flags
+ *       - modules
+ *     properties:
+ *       payment_providers:
+ *         $ref: "#/components/schemas/PaymentProvider"
+ *       fulfillment_providers:
+ *         $ref: "#/components/schemas/FulfillmentProvider"
+ *       feature_flags:
+ *         $ref: "#/components/schemas/AdminFeatureFlagsResponse"
+ *       modules:
+ *         description: The Stock Keeping Unit (SKU) code of the Inventory Item.
+ *         type: string
+ */
+export type ExtendedStore = Store & {
```

Comment:
todo: We should match the schema with the type it is describing. You would also need to update everywhere ExtendedStore was imported.
```suggestion
export type ExtendedStoreDTO = Store & {
```